src/displayapp/screens/settings/SettingSetDate.cpp

```diff
@@ -0,0 +1,272 @@
+#include "SettingSetDate.h"
+#include <lvgl/lvgl.h>
+#include <hal/nrf_rtc.h>
+#include <nrf_log.h>
+#include "displayapp/DisplayApp.h"
+#include "displayapp/screens/Symbols.h"
+
+#define POS_X_DAY   -72
+#define POS_X_MONTH   0
+#define POS_X_YEAR   72
+#define POS_Y_PLUS  -50
+#define POS_Y_TEXT   -6
+#define POS_Y_MINUS  40
+
+using namespace Pinetime::Applications::Screens;
+
+namespace {
+  static void event_handler(lv_obj_t * obj, lv_event_t event) {
+    SettingSetDate* screen = static_cast<SettingSetDate *>(obj->user_data);
+    screen->HandleButtonPress(obj, event);
+  }
+}
+
+SettingSetDate::SettingSetDate(
+  Pinetime::Applications::DisplayApp *app, Pinetime::Controllers::DateTime &dateTimeController) :
+  Screen(app),
+  dateTimeController {dateTimeController}
+{
+
+  lv_obj_t * container1 = lv_cont_create(lv_scr_act(), nullptr);
+
+  //lv_obj_set_style_local_bg_color(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(0x111111));
+  lv_obj_set_style_local_bg_opa(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, LV_OPA_TRANSP);
+  lv_obj_set_style_local_pad_all(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 10);
+  lv_obj_set_style_local_pad_inner(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 5);
+  lv_obj_set_style_local_border_width(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 0);
+  lv_obj_set_pos(container1, 30, 60);
+  lv_obj_set_width(container1, LV_HOR_RES - 50);
+  lv_obj_set_height(container1, LV_VER_RES - 60);
+  lv_cont_set_layout(container1, LV_LAYOUT_COLUMN_LEFT);
```

Comment:
Does this do anything?

Reply:
This was left from the code that I "stole" from SettingSteps. I've removed it and the code still works.